lib_json/src/main/x/json/mappings/JsonPatchOperationMapping.x

```diff
@@ -0,0 +1,47 @@
+/**
+ * A mapping for a `JsonPatch.Operation`.
+ */
+const JsonPatchOperationMapping
+        implements Mapping<JsonPatch.Operation> {
+
+    /**
+     * Construct the JsonPatchOperationMapping.
+     */
+    construct() {
+        this.typeName = $"JsonPatch.Operation";
+        this.actionMap = new HashMap();
```

Comment:
"this" is not required, since there is no name collision, but if used here should be used two line down as well. Otherwise it creates some kind of confusion to the reader